src/Runtime/OMUnique.inc

```diff
@@ -107,7 +107,7 @@ int isLessNum(void *arg1, void *arg2, OM_DATA_TYPE dataType) {
   // case ONNX_TYPE_COMPLEX64:
   // case ONNX_TYPE_COMPLEX128:
   default:
-    assert("Unsupported ONNX type in OMTensor");
+    assert(false && "Unsupported ONNX type in OMTensor");
```

Comment:
please use the recommended `llvm_unreachable`

Reply:
that was also my first thought until I realized that it's impossible because it must compile with C and cannot depend on llvm

Reply:
Got it, apologies. Should have remembered from the `.inc` file name. Please merge as is then